server/readme.md

```diff
@@ -1,12 +1,39 @@
 # U.S. Forest Service Intake Module Server
 
-For development: `npm start` and visit http://localhost:8080
+A running Postgresql database is required in order to run the server locally.  Please make sure you have installed [Postgresql](https://www.postgresql.org/) locally and created a database for this project.
+
+Then run `yarn install` to install dependencies.
+
+To setup the database run
+
+`DATABASE_URL=postgres://<user>:<pass>@localhost:<port>/<dbname> ./node_modules/sequelize-cli/bin/sequelize db:migrate`
```

Comment:
Exporting the env vars is optional.  You can put them on the command line if you want to.  BTW, the readme has had some rewriting in the flexion fork.